src/storages/storage_manager.js

```diff
@@ -25,42 +30,40 @@ const MAX_OPENED_STORAGES = 1000;
  * @property {LruCache} cache
  * @ignore
  */
-class StorageManager {
+export default class StorageManager {
```

Comment:
Watch out for this. I think we had issues with JSDoc parsing and would get e.g. `exports.Dataset` instead of `Dataset` when using it like that. Here it does not matter I guess, but somewhere else it might.

Reply:
Works just fine (even with the default export), just tried with the `BasicCrawler` locally that does have its own docs page. I guess it was some upstream bug that is now resolved.

Reply:
Ok so actually it is a problem, but only with default exports (and I guess you remember my opinion on those, meh :]).